contrib/fuzz/platforms_fuzzers.go

```diff
@@ -1,32 +0,0 @@
-//go:build gofuzz
-// +build gofuzz
-
-/*
-   Copyright The containerd Authors.
-
-   Licensed under the Apache License, Version 2.0 (the "License");
-   you may not use this file except in compliance with the License.
-   You may obtain a copy of the License at
-
-       http://www.apache.org/licenses/LICENSE-2.0
-
-   Unless required by applicable law or agreed to in writing, software
-   distributed under the License is distributed on an "AS IS" BASIS,
-   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-   See the License for the specific language governing permissions and
-   limitations under the License.
-*/
-
-package fuzz
-
-import (
-	"github.com/containerd/containerd/platforms"
-)
-
-func FuzzPlatformsParse(data []byte) int {
```

Comment:
Same question on this one.

Reply:
This one is moved under platforms/. Since Go's fuzzing support is part of testing package. I'm moving fuzzing from contrib/fuzz to each package if possible.